<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageControllers</class>
 <widget class="QWidget" name="PageControllers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ParamTable" name="paramTab"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Position Offset Calculator</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="posOffsetBox">
        <property name="prefix">
         <string>Pos Now: </string>
        </property>
        <property name="suffix">
         <string> Â°</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="posOffsetApplyButton">
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/res/+theme_light/icons/Download-96.png</normaloff>:/res/+theme_light/icons/Download-96.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParamTable</class>
   <extends>QTableWidget</extends>
   <header>widgets/paramtable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
